p4-14/v1.0.3/tex/p4.tex

```diff
@@ -658,7 +658,7 @@
 Only valid packet header fields may result in a match (when a value is
 specified for exact or ternary matches against the field), although a
 match operation may explicitly check if a header instance (or field)
-is valid. Only valid packet headers are considered for deparsing (see
+is valid (TBD: Should this be updated as asked elsewhere?). Only valid packet headers are considered for deparsing (see
```

Comment:
I will submit a PR updating this statement, to say that the fields of a search key created from header fields in invalid headers contain undefined values.